go-controller/pkg/node/openflow_manager.go

```diff
@@ -245,7 +245,38 @@ func (c *openflowManager) updateBridgeFlowCache(hostIPs []net.IP, hostSubnets []
 	}
 	dftFlows = append(dftFlows, dftCommonFlows...)
 
-	c.updateFlowCacheEntry("NORMAL", []string{fmt.Sprintf("table=0,priority=0,actions=%s\n", util.NormalAction)})
+	if config.OvnKubeNode.Mode == types.NodeModeDPU {
+		// Retrieve the local_ip for the tunnel endpoint which will be used to create OpenFlow actions
+		checkCmd := []string{
+			"get",
+			"Open_vSwitch",
+			".",
+			"external_ids:ovn-encap-ip",
+		}
+		encapIP, _, err := util.RunOVSVsctl(checkCmd...)
+		if err != nil {
+			return err
+		}
+		encapIP = strings.TrimSpace(encapIP)
+
+		// The extra actions, compared to the full mode, are due to a DHCP server on the bridge.
+		// While not impacting the other flows negatively it implies something "custom"
+		// this could be a future feature to allow users to add custom flow actions on the managed ovnkube bridge via Kubernetes API.
```

Comment:
I would remove these comments. I think the behavior of allowing broadcast to 255.255.255.255 is generic and legit in any scenario and is not "custom".

For the second flow, I think that one is DPU specific where your gateway uplink interface is not the same as LOCAL. I can see that being common across all DPUs, so we can consider that not "custom", and just part of the DPU design.